src/utils.js

```diff
@@ -383,3 +384,5 @@ export function combineURLs(baseURL, relativeURL) {
 export const isPushNotificationAvailableForUser = (state) => (
   state.pushNotification.available && state.pushNotification.deviceRegistered
 );
+
+export const getTimestampFormat = (timestamp) => moment.unix(timestamp).format('DD MMM YYYY [•] HH:mm')
```

Comment:
Should we localize this date format? This will probably be confusing for US users

Reply:
Done!